tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>Remove <code>sum, prod, min, max ...</code>  in the title. List in the text cell of <em>all</em> the supported reduction operations. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>